### PR TITLE
Add Ansible Playbook for Installation

### PR DESCRIPTION
**Pull request by katia-e**

### CODE REVIEW

File: roles/csi-driver/templates/app/demo.j2 [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (chrisvdg) [inline roles/csi-driver/templates/app/demo.j2:1]: Can be removed?

File: roles/csi-driver/templates/driver/rbac-snapshotter.j2 [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Together with the RBAC file for external-provisioner, this YAML file
```
Comment (chrisvdg) [inline roles/csi-driver/templates/driver/rbac-snapshotter.j2]: I think this file can be removed, the csi driver doesn't support snapshotting (yet)

File: roles/csi-driver/templates/namespaces/demo.j2 [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (chrisvdg) [inline roles/csi-driver/templates/namespaces/demo.j2:1]: can be removed?

File: roles/csi-driver/templates/namespaces/driver.j2 [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (chrisvdg) [inline roles/csi-driver/templates/namespaces/driver.j2:1]: Can be removed?

File: add-persistent-volume.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (chrisvdg) [inline add-persistent-volume.yml]: Do you need this on top of the file?
Also not sure about the filename

File: roles/csi-driver/templates/app/demo.j2 (hunk 1 above)
Comment (katia-e) [reply roles/csi-driver/templates/app/demo.j2:1]: every ansible playbooks and config typically starts with ---.

File: add-persistent-volume.yml (hunk 5 above)
Comment (katia-e) [reply add-persistent-volume.yml]: suggestions?

**Review by chrisvdg — APPROVED** [review]

**Pull request merged**